controllers/api/monster-routes.js

```diff
@@ -1,18 +1,18 @@
 const router = require('express').Router();
-const { Monsters } = require('../../models');
+const { Monster } = require('../../models');
 
 router.get('/:id', (req, res) => {
-    Monsters.findOne({
+    Monster.findOne({
         where: {
             id: req.params.id
         },
-        include: [
-            {
-                model: Monsters
-            }
-        ]
+        // include: [
```

Comment:
You dont need to include this since you are only calling Monster and no other table,